velox/connectors/hive/HiveConfig.h

```diff
@@ -115,13 +115,25 @@ class HiveConfig {
   static constexpr const char* kMaxCoalescedDistanceBytes =
       "max-coalesced-distance-bytes";
 
+  /// Sets the number of prefetch rowgroups
+  static constexpr const char* kPrefetchRowGroups = "prefetch-rowgroups";
+
+  /// Sets the load quantum size for a request.
+  static constexpr const char* kLoadQuantum = "load-quantum";
+
   /// Maximum number of entries in the file handle cache.
   static constexpr const char* kNumCacheFileHandles = "num_cached_file_handles";
 
   /// Enable file handle cache.
   static constexpr const char* kEnableFileHandleCache =
       "file-handle-cache-enabled";
 
+  /// set directorySizeGuess defined in dwio::common::ReaderOptions
```

Comment:
Please, provide a description that doesn't require reverse engineering the code.